src/Notepads/Controls/FindAndReplace/FindAndReplaceControl.xaml

```diff
@@ -4,143 +4,202 @@
-                <Button.KeyboardAccelerators>
-                    <KeyboardAccelerator Key="Escape"/>
-                </Button.KeyboardAccelerators>
             </Button>
         </Grid>
 
-        <Grid Row="1" Padding="1, 1, 1, 1" Height="36" x:Name="ReplaceBarPlaceHolder">
+        <Grid Grid.Row="1" Grid.Column="2" Padding="1, 1, 1, 1" Height="36">
 
             <Grid>
                 <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="*" />
                     <ColumnDefinition Width="Auto" />
                     <ColumnDefinition Width="Auto" />
                 </Grid.ColumnDefinitions>
 
-                <Grid Column="0">
-                    <TextBox x:Name="ReplaceBar"
-                             x:Uid="FindAndReplace_ReplaceBar"
-                             Style="{StaticResource TransparentTextBoxStyle}"
-                             Foreground="{ThemeResource SystemControlHighlightChromeAltLowBrush}"
-                             HorizontalAlignment="Stretch"
-                             VerticalAlignment="Stretch"
-                             IsSpellCheckEnabled="False"
-                             TextAlignment="Left"
-                             BorderThickness="0"
-                             FontSize="15"
-                             TextChanged="ReplaceBar_OnTextChanged"
-                             KeyDown="ReplaceBar_OnKeyDown"
-                             GotFocus="ReplaceBar_GotFocus"/>
-                </Grid>
-
                 <Button x:Name="ReplaceButton"
                         x:Uid="FindAndReplace_ReplaceButton"
                         IsEnabled="False"
                         IsTabStop="False"
                         Width="36"
-                        Grid.Column="1" 
+                        Grid.Column="0" 
                         FontSize="16"
                         Style="{StaticResource DismissButtonStyle}"
                         Content="&#xE8AB;"
                         FontFamily="Segoe MDL2 Assets"
                         Click="ReplaceButton_OnClick">
+                    <Button.KeyboardAccelerators>
+                        <KeyboardAccelerator Modifiers="Menu" Key="R"/>
+                    </Button.KeyboardAccelerators>
                 </Button>
 
                 <Button x:Name="ReplaceAllButton"
                         x:Uid="FindAndReplace_ReplaceAllButton"
                         IsEnabled="False"
                         IsTabStop="False"
                         Width="36"
-                        Grid.Column="2" 
+                        Grid.Column="1" 
                         FontSize="16"
                         Style="{StaticResource DismissButtonStyle}"
                         Content="&#xE7FD;" 
                         FontFamily="Segoe MDL2 Assets"
                         Click="ReplaceAllButton_OnClick">
+                    <Button.KeyboardAccelerators>
+                        <KeyboardAccelerator Modifiers="Menu" Key="A"/>
```

Comment:
Alt+A is reserved by Window clipping tool. Ctrl+shift+enter is good enough here.

Reply:
By window clipping tool do you mean snipping tool?? I can't find when it uses this shortcut. I used "Alt+A" cause notepad uses it. "Ctrl+Shift+Enter" was just to match funtionality with VS Code. Wouldn't users expect "Alt+A" to do this by default??

Reply:
Oh, I see. Actually it was reserved by another program that I am using. Nvm.

Reply:
Should I change it??

Reply:
Let's just remove alt+A since notepad is too old. Let's follow the trend of VS Code here.

Reply:
Just noticed that notepad++ and vs studio use "Alt+A" as well, may be other text editors do this too.

Reply:
Let's have Ctrl+Shift+Enter for now and see how user response. Most of the people don't use shortcut at all. We should be fine.

Reply:
I have removed Alt+A, so it is ready to be merged.